opentech/apply/dashboard/tables.py

```diff
@@ -0,0 +1,12 @@
+import django_tables2 as tables
+from opentech.apply.funds.models import ApplicationSubmission
+
+
+class DashboardTable(tables.Table):
+    submit_time = tables.DateColumn(verbose_name="Submitted")
+    page = tables.Column(verbose_name="Fund")
+
+    class Meta:
+        model = ApplicationSubmission
+        fields = ('title', 'page', 'submit_time')
```

Comment:
currently the title column outputs a dash regardless of the submission

Reply:
Fixed in 75dc7ea

Reply:
👍